js/account-popover.jquery.js

```diff
@@ -161,13 +166,8 @@ class AccountPopover {
     }
 
     // Click away to dismiss
-    $('html').on('click.popoverDismiss', (e) => {
-      const $target = $(e.target);
-      if ($target.closest('.popover').length === 0 && $target.closest('.use-popover').length === 0 && $target.closest($trigger).length === 0) {
```

Comment:
Seem to have lost this check - I think its behaviour was designed so that clicking on the popover itself doesn't close the popover

Reply:
Sure enough on the [preview site](https://deploy-preview-235--id7.netlify.app/) if you click on the popover (like on the grey area surrounding the bottom buttons) it now closes itself

Reply:
Updootled